index.html

```diff
@@ -0,0 +1,1273 @@
+                         <br>
+                         <br>Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an 
+                         <br>unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply 
+                         <br>dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of 
+                         <br>type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.
+                     </p>
+
+                 <img src="media/closemodaldesktop.png" class="modal-close" href="#">
+
+                 <img src="media/htmlicon.png" class="cajahtml">
+
+                 <img src="media/css.png" class="cajacss">
+
+                 <img src="media/javascripticon.png" class="cajajavascript">
+
+                 <img src="media/rubyicon.png" class="cajaruby">
+
+                 <img src="media/bootstrapicon.png" class="cajabootstrap">
+
+                 <img src="media/linea.png" class="linea">
+
+                 <button type="button" class="seelive" >See live</button>
+
+                 <img src="media/boton1.png" class="fader">
+
+                 <button type="button" class="seesource">See Source</button>
+
+                 <img src="media/botongithub.png" class="githubicon">
+        </div>
+
+     </section>
+
+     <!--SEGUNDA CARD ESCRITORIO-->
+
+     <section class="modal1">
+
+     <div class="modal_container">
+
+         <ul>
+            
+             <li id="canopyesc1">
+
+                <p>FACEBOOK</p> 
+
+             </li>
+
+             <li class="punto3">
+                 <img src="media/punto.png" >
+             </li>
+
+             <li class="backend1">
+                 <p>full Stack Dev</p>
+             </li>
+
+             <li class="punto4">
+                 <img src="media/punto.png">
+             </li>
+
+             <li class="año1">
+                 <p>2015</p>
+             </li>
+
+         </ul>
+
+             <img src="media/escritoriotrabajo2.png" class="modal_img">
+
+             <h2 class="modal_title1">Multi-Post Stories</h2>
+
+                 <p class="modal_paragraph">
+                     Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been 
+                     <br>the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                     <br>galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and 
+                     <br>typesetting industry. 
+                     <br>
+                     <br>Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an 
+                     <br>unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply 
+                     <br>dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of 
```

Comment:
- [ ] Great job working on this milestone. As per project requirements you are supposed to implement a javascript object to store data about each project and not by adding the data in HTML.
![image](https://github.com/maximoortelli/Portfolio-reactivo/assets/62849872/13457c83-1226-4160-9730-085b83a9769f)
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

- The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.